app/internal/agenda_events.py

```diff
@@ -89,3 +90,22 @@ def get_events_in_time_frame(
     """Yields all user's events in a time frame."""
     events = get_all_user_events(db, user_id)
     yield from filter_dates(events, start_date, end_date)
+
+
+def make_dict_for_graph_data(db: Session, user_id: int) -> Dict:
```

Comment:
Specify what types the `Dict` contains: `Dict[key, value]`

Reply:
done